service/src/main/java/bio/terra/profile/service/spendreporting/azure/model/SpendDataItem.java

```diff
@@ -0,0 +1,4 @@
+package bio.terra.profile.service.spendreporting.azure.model;
+
+public record SpendDataItem(
+    String resourceType, Double cost, String currency, SpendCategoryType spendCategoryType) {}
```

Comment:
Since we're reporting monetary values, seems like `cost` should be a BigDecimal rather than a double.